zerver/webhooks/stripe/view.py

```diff
@@ -198,6 +186,40 @@ def default_body(update_blacklist: List[str]=[]) -> str:
             amount=amount,
             end=end
         )
+
+    if category.startswith('invoice'):
+        INVOICE_INFO = {
+            "created": "has been created",
+            "deleted": "has been deleted",
+            "finalized": "has been finalized",
+            "marked_uncollectible": "has been marked as uncollectible",
+            "payment_failed": "has failed",
+            "payment_succeeded": "has succeeded",
+            "sent": "has been sent",
+            "upcoming": "has been created",
+            "updated": "has been updated",
+            "voided": "has been voided"
+        }
+        object_id = object_['id']
+        link = "https://dashboard.stripe.com/invoices/{}".format(object_id)
+        amount = amount_string(object_["amount_due"], object_["currency"])
+        body_t = "An {invoice_type} **[invoice]({link})** for the payment of amount **{amount}** {end}."
+
+        end = INVOICE_INFO.get(event)  # type: ignore # expression has type Optional[str]
```

Comment:
This line will only be called if `category` starts with `invoice`. So this code should work regardless of where `INVOICE_INFO` is defined because you are only calling it for a specific event type. Does that make sense?